src/components/createAppointment/MemberDetail.tsx

```diff
@@ -0,0 +1,74 @@
+import { Avatar, Chip, Typography, Stack } from "@mui/material";
+import MaleIcon from "@mui/icons-material/Male";
+import FemaleIcon from "@mui/icons-material/Female";
+import TransgenderIcon from "@mui/icons-material/Transgender";
+import CakeIcon from "@mui/icons-material/Cake";
+import { User } from "@/types/User";
+import { Gender } from "enum/gender";
+
+type props = {
+  value: User | null;
+};
+
+const profile_layout = {
+  width: "50vw",
+  minWidth: "200px",
+};
+const stack_design = {
+  padding: "30px",
+  width: "400px",
+  height: "50%",
+  boxSizing: "border-box",
+};
+
+const avatar = { width: 120, height: 120 };
+
+export default function MemberDetail(props: props) {
+  return (
+    <>
+      {props.value && (
```

Comment:
There's a little box appear when you hover the memberdetail card. Can you fix it?
<img width="532" alt="image" src="https://user-images.githubusercontent.com/62459518/224677281-7d43e7a9-380f-4697-9f95-86415a1e9c3f.png">

Reply:
Fixed